src/Microsoft.AspNet.Routing/Template/TemplateBinder.cs

```diff
@@ -132,25 +141,22 @@ public TemplateValuesResult GetValues(RouteValueDictionary ambientValues, RouteV
 
             // Any default values that don't appear as parameters are treated like filters. Any new values
```

Comment:
Yeah, I still don't really see a reason to. No one understands this feature anyway.